docs/release/trg-0/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Documentation Architecture
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft - MISSING EXTERNAL INFORMATION  | 10-Jan-2024  | Transfered from Release Acceptance Criteria into TRG|
```

Comment:
What external info is missing?
"Initial version" as description would be better IMO.

Reply:
Initial version
21-Mar-2024
Transferred...